spec/support/application.rb

```diff
@@ -12,6 +12,14 @@ def fixture_file(file, options = {})
         Rails.root.join(*path)
       )
     end
+
+    def with_env(env)
+      @original_env = ENV['ENV']
```

Comment:
Would it be preferable to use `Rails.env` instead of accessing the environment variables in this way? It would also violate [the Rails/EnvironmentVariableAccess cop](https://docs.rubocop.org/rubocop-rails/2.10/cops_rails.html#railsenvironmentvariableaccess), although we currently override the cop so that we are allowing reads but not writes ([here](https://github.com/ministryofjustice/laa-court-data-ui/blob/fea27b2407dda7a6abface997359b70b248e97ad/.rubocop.yml#L102)).